example/App.js

```diff
@@ -72,6 +97,13 @@ const App = React.createClass({
   },
 
   render() {
+    if (this.state.useGoogleMaps) {
+      return this.renderExamples([
+        [EventListener, 'Events'],
```

Comment:
How about instead of duplicating this, we just add a third element to the array which is a boolean indicating if Google Maps is supported.